Dockerfile

```diff
@@ -61,4 +61,4 @@ VOLUME ["/code/bakerydemo/media/images/"]
 ENTRYPOINT ["/code/docker-entrypoint.sh"]
 
 # Start uWSGI
-CMD ["uwsgi", "/code/etc/uwsgi.ini"]
+CMD ["/venv/bin/uwsgi", "/code/etc/uwsgi.ini"]
```

Comment:
Issue: This change shouldn't be necessary, as `/venv/bin` is added to the `PATH` [here](https://github.com/wagtail/bakerydemo/blob/97624f71a96c12e12150cb497cc7b517ed5dd218/Dockerfile#L22)